packages/design-tokens/src/themes/heart.ts

```diff
@@ -244,7 +244,7 @@ export const heartTheme: Theme = {
       fontWeight: 400,
       fontSize: "1.25rem",
       lineHeight: "1.875rem",
-      letterSpacing: "-0.5px",
+      letterSpacing: "0px",
```

Comment:
```suggestion
      letterSpacing: "0",
```

Usually when anything is `0` you don't put a measurement on it :)